api/src/test/java/org/openmrs/ObsTest.java

```diff
@@ -311,7 +311,13 @@ public void isComplex_shouldReturnTrueIfTheConceptIsComplex() throws Exception {
 	 */
 	@Test
 	public void setValueAsString_shouldFailIfTheValueOfTheStringIsEmpty() throws Exception {
+		ConceptDatatype cdt = new ConceptDatatype();
+		cdt.setHl7Abbreviation("ST");
+		Concept c = new Concept();
+		c.setDatatype(cdt);
```

Comment:
These tests weren't actually testing the correct thing... they were failing because the concept was null, not because the value was null